modules/afsocket/socket-options-inet.h

```diff
@@ -36,6 +36,8 @@ typedef struct _SocketOptionsInet
   gint tcp_keepalive_intvl;
   gint tcp_keepalive_probes;
   gchar *interface_name;
+  /* calculated */
+  gboolean multicast;
```

Comment:
From the perspective of `SocketOptionsInet`, it's an option that has to be set by the user of `SocketOptionsInet`, just like any other options. I think we shouldn't note here that the value is calculated from something unknown to this class. I'd also remove `/* user settings */`.

The best would be if this flag was removed from the struct, and we could pass it as a parameter of `socket_options_inet_setup_socket()`. However, this is not possible, because `socket_options_inet_setup_socket()` is virtual and isn't called directly in `AFInetDestDriver` (and `multicast` is meaningless at the abstract `SocketOptions` level); so I think it is the good place.